src/modules/common/components/landing.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { loginToPanoptes } from '../../../services/panoptes';
+
+const SignedOut = () =>
+  <div>
+    <div className="landing__buttons">
+      <button className="button button--full-teal landing__button" onClick={loginToPanoptes}>Sign In</button>
+      <br />
+      <a className="button button--full-teal landing__button" href="https://www.zooniverse.org/projects">Back to Classifying</a>
+    </div>
+    <div className="landing__links-section">
+      <p className="landing__links-heading">Quick Links</p>
+      <a className="landing__link" href="https://www.zooniverse.org/help">Project Builder Help</a>
+      <a className="landing__link" href="https://www.zooniverse.org/help/lab-policies">Project Builder Policies</a>
+      <a className="landing__link" href="https://www.zooniverse.org/lab-best-practices/introduction">Best Practices Guide</a>
+      <a className="landing__link" href="https://www.zooniverse.org/about/acknowledgements">Acknowledgements</a>
+      <a className="landing__link" href="https://www.zooniverse.org/talk/18">Project Builder Talk</a>
+      <a className="landing__link" href="https://www.zooniverse.org/help/glossary">Glossary</a>
+    </div>
+  </div>;
+
+const SignedIn = () =>
+  <div>
+    <div className="landing__row">
+      <div>
+        <h4>Projects</h4>
+        <p className="landing__row-description">To build a project just upload your data and choose the tasks you want the volunteers to do. To find out more, read our <a href="https://www.zooniverse.org/lab-how-to">How to Build a Project documentation</a>, or click the button to get started.</p>
+      </div>
+      <div>
+        <a href="https://www.zooniverse.org/lab"className="button button--full-teal landing__button">Build a Project</a>
+      </div>
+    </div>
+    <div className="landing__row">
+      <div>
+        <h4>Organizations</h4>
+        <p className="landing__row-description">
```

Comment:
I intended `row-description` to be an element, but that's not a good label. Renamed to `paragraph` for now, though I'm not wild about that either. I think based on BEM guidelines it should just be a `block-element` class name (no modifier), but I'm not sure how to refer to the element that's like "the paragraph/description for a row" or "content related to a call to action button."